trin-cli/src/dashboard/grafana.rs

```diff
@@ -0,0 +1,100 @@
+use base64;
+use nanotemplate::template;
+use serde::Deserialize;
+use std::fs;
+use ureq;
+
+const TEMPLATE_FILENAME: &str = "./trin-cli/src/dashboard/metrics_dashboard.json.template";
```

Comment:
Is this path safe under Windows? I think we should move to use [Path](https://doc.rust-lang.org/stable/std/path/struct.Path.html) instead of strings for folder paths.